server/models/meetup.model.js

```diff
@@ -36,9 +36,9 @@ var meetupSchema = Schema({
       ]
     }
   ],
-  subscriber:[
+  subscribers:[
     {
-      userID: ObjectId,
+      userID: String,
```

Comment:
Why would you use type String for userID?

If we need the user model within meetup model, we should do something along the lines of 


`
userID: {
        type:ObjectId ,
        ref:'Users'
    }
`

Reply:
@afirdousi Ah yes, will do that.